git/compat.py

```diff
@@ -8,6 +8,7 @@
 # flake8: noqa
 
 import sys
+import six
```

Comment:
I think I looked very carefully, but couldn't find a new dependency. Is six available in Py2.7 and onwards by default ?